FND/Parties/Parties.rdf

```diff
@@ -113,26 +96,23 @@
 	
 	<owl:Class rdf:about="&fibo-fnd-pty-pty;IndependentParty">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-aap-agt;AutonomousAgent"/>
-		<rdfs:label>independent party</rdfs:label>
-		<owl:disjointWith rdf:resource="&fibo-fnd-pty-rl;AgentInRole"/>
-		<skos:definition>any person or organization</skos:definition>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-fnd-pty-pty;OrganizationMember">
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-pty-pty;PartyInRole"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-pty-rl;isPlayedBy"/>
-				<owl:someValuesFrom>
-					<owl:Restriction>
-						<owl:onProperty rdf:resource="&lcc-lr;isMemberOf"/>
-						<owl:allValuesFrom rdf:resource="&fibo-fnd-org-org;Organization"/>
-					</owl:Restriction>
-				</owl:someValuesFrom>
+				<owl:onProperty rdf:resource="&fibo-fnd-pty-pty;hasMailingAddress"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label>organization member</rdfs:label>
-		<skos:definition>identifies an organization or someone that plays a membership role in some organization</skos:definition>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasAddress"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>independent party</rdfs:label>
+		<owl:disjointWith rdf:resource="&fibo-fnd-pty-rl;AgentInRole"/>
```

Comment:
No! This utterly screws up legitimate implementations already in production and is a show stopper for me. If you want it for testing the ontology then add it temporarily.

Reply:
> No! This utterly screws up legitimate implementations already in production and is a show stopper for me. If you want it for testing the ontology then add it temporarily.

As I said, it's been there for years, literally.  Most FIBO users don't run reasoners in operational / production settings, only when testing the ontology and looking for issues while it is under development or I'm certain you would have found it ages ago.